pre-commit

```diff
@@ -10,6 +10,8 @@ if [ -L "$0" ]; then
 	else
 		DEV_LIB_PATH=$( dirname $( readlink "$0" ) )
 	fi
+elif [ -e "$(dirname "$0")/.dev-lib-path" ]; then
+    DEV_LIB_PATH="$(realpath $(cat "$(dirname "$0")/.dev-lib-path"))"
 else
```

Comment:
Thanks for the tips, I'm mostly trying to overcome the need for symlink, might need to see if `realpath` is really necessary, and probably someway to get powershell/cmd to work as well, as I'm only building for Git bash here, which has limited symlinking ability.